Segmentation fault when running gosec with go 1.13

### Summary 

If `gosec` command is run on a Golang 1.13 environment, a runtime error is tiggered

```
$ gosec
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x6f4d18]

goroutine 1 [running]:
main.prepareVersionInfo(...)
	/go/src/github.com/securego/gosec/cmd/gosec/version.go:23
main.main()
	/go/src/github.com/securego/gosec/cmd/gosec/main.go:244 +0x1108

```
### Steps to reproduce the behavior
Easiest way for me is using docker golang:1.13 image and, inside it, just run `gosec`.

### gosec version
No version specified, so latest one in repository

### Go version (output of 'go version')
`go version go1.13.12 linux/amd64`

### Operating system / Environment
Docker image `golang:1.13`

### Expected behavior
Gosec executed with normallity
### Actual behavior
Segmentation fault, pointing to the version info retrieval system

--- a/cmd/gosec/version.go
+++ b/cmd/gosec/version.go
@@ -1,9 +1,5 @@
 package main
 
-import (
-	"runtime/debug"
-)
-
 // Version is the build version
 var Version string
 
@@ -19,7 +15,10 @@ var BuildDate string
 // https://github.com/golang/go/issues/29814 gets resolved.
 func prepareVersionInfo() {
 	if Version == "" {
-		bi, _ := debug.ReadBuildInfo()
-		Version = bi.Main.Version
+		// bi, _ := debug.ReadBuildInfo()
+		// Version = bi.Main.Version
+		// TODO use the debug information when it will provide more details
+		// It seems to panic with Go 1.13.
+		Version = "dev"
 	}
 }

